source/android/mobile/licenses/OFL.txt

```diff
@@ -0,0 +1,93 @@
+Copyright 2018 The Beth Ellen Project Authors (https://github.com/googlefonts/BethEllen)
+
+This Font Software is licensed under the SIL Open Font License, Version 1.1.
+This license is copied below, and is also available with a FAQ at:
```

Comment:
this feels like a reasonable license, but it's not on the list of generally approved licenses and I'm not a lawyer :)

are there fonts that are MIT/BSD licensed we could use? if not, we might have to got through an approval process...

Reply:
we went through the approval process and didn't receive a negative answer from the reviewers, so I assume it succeeded